2018-04-05T11:12:20.796Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '10717' in collection with ID 'Asset:org.bjitgroup.com.Talktime' does not exist)
    at /home/tasmin/.nvm/versions/node/v8.10.0/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-04-05T11:12:20.798Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '10717' in collection with ID 'Asset:org.bjitgroup.com.Talktime' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '10717' in collection with ID 'Asset:org.bjitgroup.com.Talktime' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/tasmin/.nvm/versions/node/v8.10.0/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-04-05T11:15:04.558Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'Asset:org.bjitgroup.com.TalkTime' in collection with ID '$sysregistries' does not exist)
    at /home/tasmin/.nvm/versions/node/v8.10.0/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-04-05T11:15:04.559Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'Asset:org.bjitgroup.com.TalkTime' in collection with ID '$sysregistries' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'Asset:org.bjitgroup.com.TalkTime' in collection with ID '$sysregistries' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/tasmin/.nvm/versions/node/v8.10.0/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-04-05T11:25:45.176Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'Asset:org.bjitgroup.com.TalkTime' in collection with ID '$sysregistries' does not exist)
    at /home/tasmin/.nvm/versions/node/v8.10.0/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-04-05T11:25:45.176Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'Asset:org.bjitgroup.com.TalkTime' in collection with ID '$sysregistries' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'Asset:org.bjitgroup.com.TalkTime' in collection with ID '$sysregistries' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/tasmin/.nvm/versions/node/v8.10.0/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-04-05T11:37:15.554Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '10717' as the object already exists)
    at /home/tasmin/.nvm/versions/node/v8.10.0/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-04-05T11:37:15.555Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '10717' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '10717' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/tasmin/.nvm/versions/node/v8.10.0/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
